figlet.yaml

```
@@ -0,0 +1,61 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: figlet
+  version: 2.2.5
+  epoch: 27
+  description: Program that creates large characters out of ordinary screen characters.
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cmatsuoka/figlet
+      branch: master
+      expected-commit: 202a0a8110650a943f1125f536b3bb455cf72ee1
+
+  - uses: autoconf/make-install
+    with:
+      opts: "DEFAULTFONTDIR=/usr/share/figlet/fonts DESTDIR=${{targets.destdir}} BINDIR=/usr/bin MANDIR=/usr/share/man"
```

```suggestion
      opts: "DEFAULTFONTDIR=/usr/share/figlet/fonts BINDIR=/usr/bin MANDIR=/usr/share/man"
```

DESTDIR set already: https://github.com/chainguard-dev/melange/blob/3e035809d17ac5761e3c3be0cd8b8916c748c299/pkg/build/pipelines/autoconf/make-install.yaml#L20